src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int maxRandomValue = 100;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball newBall = new Ball();
+        newBall.setColor(colorSupplier.getRandomColor());
```

Comment:
The `setColor` method is currently being passed a `String` from `colorSupplier.getRandomColor()`. Once the `ColorSupplier` class is updated to return a `Color` type, ensure that the `setColor` method in the `Ball` class is updated to accept a `Color` type instead of a `String`.